src/Middleware/NodeServices/src/INodeServices.cs

```diff
@@ -9,6 +12,7 @@ namespace Microsoft.AspNetCore.NodeServices
     /// might change over time (e.g., the process might be restarted), the <see cref="INodeServices"/> instance
     /// will remain constant.
     /// </summary>
+    [Obsolete("Use Microsoft.AspNetCore.SpaServices.Extensions")]
```

Comment:
There is no `INodeServices` in `Microsoft.AspNetCore.NodeServices` - what are consumers supposed to be use instead?